src/FsAutoComplete.Core/Decompiler.fs

```diff
@@ -17,30 +17,35 @@ let private tryGetValue key (dict:IReadOnlyDictionary<_,_>) =
     | true, x -> Some x
     | false, _ -> None
 
-type OutputPositions = 
+type OutputPositions =

```

Comment:
instead of a single case DU, you can use a type

```fsharp
type OutputPositions =
   {  MethodsPosition:IReadOnlyDictionary<MethodDef, Position>	          
       MethodsPosition:IReadOnlyDictionary<MethodDef, Position> 
       FieldsPosition:IReadOnlyDictionary<FieldDef, Position> 		          
       FieldsPosition:IReadOnlyDictionary<FieldDef, Position> 
       PropertiesPosition:IReadOnlyDictionary<PropertyDef, Position> 		          
       PropertiesPosition:IReadOnlyDictionary<PropertyDef, Position> 
       EventsPosition:IReadOnlyDictionary<EventDef, Position> 		           
       EventsPosition:IReadOnlyDictionary<EventDef, Position> 
       TypesPosition:IReadOnlyDictionary<TypeDef, Position> 		        
       TypesPosition:IReadOnlyDictionary<TypeDef, Position> }
```

this will cleanup the code later from 

```fsharp
   member this.TryFindMethod def =
        let (OutputPositions (methodsPosition=methodsPosition)) = this
         methodsPosition |> tryGetValue def		          methodsPosition |> tryGetValue def
```

to just

```fsharp
   member this.TryFindMethod def =
         def.MethodsPosition |> tryGetValue def
```